src/storage.rs

```diff
@@ -153,33 +153,28 @@ impl Storage {
         let ret = self
             .0
             .get(&key)
-            .ok_or_else(|| {
-                bail_context!(
-                    "Could not find {:?} when getting from storage",
-                    storable_index
-                )
-            })?
+            .ok_or_else(|| InternalError::StorageItemNotFound(format!("{:?}", storable_index)))?
```

Comment:
we could make a wrapper function for this error just like the other one. The signature would look like:
```rust
fn storage_item_not_found(d: impl Debug) -> InternalError {
    InternalError::StorageItemNotFound(format!("{:?}", d))
}
```

Reply:
I actually just ended up simplifying the StorageItemNotFound error to not take a string, since in at least one instance it would have to print a fairly large string. If the additional information is needed, it's probably more appropriate to put it in a log